thread/src/components/Logo.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
```

Comment:
Move this whole file to common, on my pr and doms, we have created a index file so it can be exported as part of one big object